.github/workflows/ruby-build.yml

```diff
@@ -235,54 +229,3 @@ jobs:
         shell: bash
         run: |
           codeql database analyze --search-path "${{ runner.temp }}/ruby-bundle" --format=sarifv2.1.0 --output=out.sarif ../database ruby-code-scanning.qls
-
```

Comment:
Let's keep this test for now, I'd like to be sure that things keep working after dropping `cross`.

Reply:
We can't keep it here - the ruby extractor built by the test workflow here is no longer compatible with centos 7, as it's not built against centos-7 with cross, nor is it compiled against our hermetic toolchain. I can check internally if we have a better place for this (or a similar) test, I think we do.